hicommonwealth/commonwealth
# Fix `check-types` in `packages/commonwealth/`

## Link to Issue
Closes: #6265

## Description of Changes
- Updates `include` in `commonwealth/tsconfig.json` so that the client, tests, and scripts are all included. Unfortunately, this means they are included in the final build but at the moment this is the best solution since the `tsc` `--project` option is incompatible with `--build`.

## "How We Fixed It"

Reverts the removal of directories from the main tsconfig `include` array.

## Test Plan
In the commonwealth package:
- Make a change on the client that would clearly cause a type error and run `yarn check-types`
  - Ensure this throws an error

## Deployment Plan

1. 

## Other Considerations

-
